dm/worker/worker.go

```diff
@@ -69,6 +71,7 @@ func NewWorker(cfg *Config) (*Worker, error) {
 		relayHolder: NewRelayHolder(cfg),
 		tracer:      tracing.InitTracerHub(cfg.Tracer),
 		subTasks:    make(map[string]*SubTask),
+		l:           log.With(zap.String("component", "worker controller")),
```

Comment:
seems have `component`, `unit`, `feature` three levels, can you describe the difference between them？